baybe/utils/dataframe.py

```diff
@@ -413,3 +414,42 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_printing_dataFrame(df: pd.DataFrame) -> str:
+    """Return a pretyt/readable str Object.
+
+    Args:
+        df: The dataframe to be printed.
+
+    Returns:
+        The values to be printed as a str table.
+    """
+    # select the first 10 rows/values and store them in a str table
+    # TODO: remove hard coded numbers and allow user defined specifications
```

Comment:
I guess we agree that "pretty printing" a dataframe means to just print a reduced number of rows and columns. Do we want to have these numbers fixed, and if so, which number should we choose? I think the choices made by @RimRihana  here are reasonable as defaults, and I am having trouble imagining a good way how a user could specify it.
@Scienfitz @AdrianSosic please also share your opinions here.

Reply:
See my comment regarding the `Settings`class (i.e., I don't think we need to bother with customization in this PR). I  think what pandas does by default is not too bad. 

<img width="1031" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/07e1467f-cb50-4ec5-9fe9-82b2117096dc">

The only thing I'd like to avoid (which I've seen in some contexts) is to get a "line wrap" in the print

Reply:
@RimRihana did you try this: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.option_context.html

using building pandas functionality should be the first approach if 

it seems theres not much of a wishlist and i owuld also agree that simply this is it:
- limit num rows and num cols
- avoid linewrap

Reply:
And here are some more infos on options in case it is helpful: https://pandas.pydata.org/pandas-docs/stable/user_guide/options.html#options